rust/libnewsboat/src/utils.rs

```diff
@@ -1179,4 +1198,11 @@ mod tests {
         let input = expected.to_owned() + "# and a comment";
         assert_eq!(strip_comments(&input), expected);
     }
+
+    #[test]
+    fn t_extract_filter() {
+        let expected = ("~/bin/script.sh", "https://newsboat.org");
+        let input = "filter:~/bin/script.sh:https://newsboat.org";
+        assert_eq!(extract_filter(input), expected);
```

Comment:
Please port new C++ tests to Rust. The goal is to have the same tests on both side, so that:
1. bindings are tested by C++ tests
2. when we eventually drop the bindings and their C++ tests, we still have a complete test suite on the Rust side.

Reply:
Okay

Reply:
Um, but you added it only on the Rust side? Despite the fact that Rust and C++ suites should be equal?

Reply:
Oh, yes.